ref/undescribedfaq.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 'FAQ About Undescribed Galls'
+date: '2022-05-23'
+description: 'What to do if you find an undescribed gall.'
+author:
+    name: Adam Kranz
+---
+At the time of writing, of the 2806 galls listed on the Gallformers database, 963 are undescribed. That ratio has not changed significantly since the early days of the site and isn't likely to change much in the future. 
+You might reasonably wonder, if a gall can be distinctly identified by its morphology and host plant, such that we can represent it with its own Gallformers entry and consistently apply a Gallformers Code to observations, in what sense is it "undescribed", and what would it take to change that?
+A gall is part of the extended phenotype of the inducing organism, and in many cases the traits of the gall are sufficient to identify the species that induced it. The traits of the gall can often even be used to make an educated guess about what other species the inducer is related to. 
+Regardless, the rules of taxonomy dictate that in order to formally describe a new species, taxonomists need to have a physical specimen of the inducing organism in front of them. The species isn't considered "described" until their description is published in a peer-reviewed academic journal (so species described in grad school theses but never published in a journal are still considered undescribed). 
+One of our main goals at Gallformers is to facilitate you, an amateur or academic naturalist, in the process of collecting and rearing such a specimen and getting it in front of an appropriate taxonomist. 
+
+A note to preface: Raising an inducer is not hard, but each individual attempt has low odds of success. The biggest reason experts are more likely to succeed is not any particular technique but because they search harder and collect galls in larger numbers than any individual amateur. As a community, we can distribute that effort across many observers. So while you might not be successful yourself, you are still contributing to the process that builds the knowledge we need for someone to eventually succeed. 
+
```

Comment:
# Being Prepared

When in the field looking for undescribed galls you need to be prepared if you want to maximize your chances of success in collecting and rearing. Bringing a couple of basic tools with you will greatly improve your odds.

- Something to section galls. e.g., a small sharp pocket knife
- Containers for collection. The easiest and most readily available are zip-top plastic bags
- A way to capture the details of the collection. Could be a marker to write on the collection bag, a field notebook, voice memos on your phone, etc

When you collect a gall it is critically important that you capture several pieces of information. Without this information the specimen can be useless:

- Date and Time of Collection (make sure to specify the timezone!)
- Location of collection, Lat/Long if you have GPS, if not then the locale info and a rough description of the location
- Host plant species

## Sectioning Galls

In trying to determine the taxa of unknown galls it is often necessary to section them. This can be done in the field or after collection. In most cases sectioning will destroy the gall so make sure that you are sectioning it for the right reasons. There is more info about why and when to section galls later in this FAQ.

When sectioning a gall it is advisable to start with a cut well of the mid-line of the gall. This is because in many cases the larvae are found at the center of the gall and this reduces the chances that you will destroy the larvae.

Reply:
This is just a rough outline of what I think we need. Add whatever other info and editorial changed you want.